tests/integration/security/authorization_test.go

```diff
@@ -662,169 +662,156 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.egress-gateway").
 		Run(func(t framework.TestContext) {
-			ns := namespace.NewOrFail(t, t, namespace.Config{
-				Prefix: "v1beta1-egress-gateway",
-				Inject: true,
-			})
+			ns := apps.Namespace1
+			rootns := newRootNS(t)
+			src0 := apps.A.Match(echo.Namespace(apps.Namespace1.Name()))
+			src1 := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+			for _, dst := range []string{util.CSvc, util.VMSvc} {
+				t.NewSubTest(fmt.Sprintf("to %s/", dst)).Run(func(t framework.TestContext) {
+					args := map[string]string{
+						"Namespace":     ns.Name(),
+						"RootNamespace": rootns.rootNamespace,
+						"src":           util.ASvc,
```

Comment:
can we just use `b` and `c` in the code and also the policy template, you can pass `vm` to `b` without problems.

Reply:
Sure, updated.